.travis.yml

```diff
@@ -227,34 +227,35 @@ script:
         -DWITH_TOKUDB=ON
       ";
       if [[ "$INVERTED" == "ON" ]]; then
-        CMAKE_OPT+=" -DWITH_NUMA=OFF";
+        CMAKE_OPT+="
```

Comment:
Please, do not do this `.travis.yml` refactoring in this PR. Because otherwise 5.5 branch won't have it.
Just add `-DWITH_INNODB_MEMCACHED=ON` here.